### ISSUE TITLE
Inverted fermata

### ISSUE THREAD

**Issue by maccallum** [HfMT-ZM4/musicxml2jmsl#7]
JMSL MARK 21

**Comment by computermusik** [comment]
Hi John,

Did you already push the new version that contains the change?

Thanks,

Georg

**Comment by maccallum** [comment]
Looks like I had forgotten to push that, but it's there now

**Comment by computermusik** [comment]
Hmm, it's still not doing the inverted fermatas.

Can you check?

Thanks,

Georg

**Comment by maccallum** [comment]
Ok, there was a bug that I just fixed and pushed, so it's putting the right mark (21) in the right place.

There's potentially something wrong on my machine though. The stems aren't displaying in the correct direction, so stems that should be inverted are not. I haven't changed anything related to stems in weeks, so I'm wondering if this has anything to do with the new jar files I installed the other day? If you notice anything wrong in the JMSL files, file an issue and I'll take a look asap.

I'll be offline for most of the rest of the night, but I can look at this again in the morning.

Best,
John
